asset/404-img.svg

```diff
@@ -1 +1,3 @@
Y5n2YzRnEqmPbyf28m7JERNkvgXTo5ag8RfQ+xbat21599rlnL8tmsz/CNhhBgCY8TMZFxj0wMACQBmLmYr4noj33kJBijSJRS0ItEGvepY00zM/PBw4FZ+gy0AHxwLat25QJTyF2qGQQBSmxbeu2R2u12jWdMlTUljAQAIUS5vSgeY+10EBmz+sMLAPYHJnZLpfPES0QBSveV4xCCYue/orUi8XQq6Y8NPVbTnIOJlg0cwgJ0BJoyvMXTIro3UNSovcKtm3ddn+9Xr+9Uql0tlSOQ/wvyMhNyyQCChk8S8xNUSElE1oFUjaXJflauqaTMHDUBoNG3TGg6a5LoYSh+9ds27qttbSYnJA3wZYfoKW7AVpJavJ4aeyKLcyqoxJsVyDRVyC2bd12w7PPPQsLCwufzmZzkEoFV/KRkWNgBEaxoXkPBRS+V2/UiYlPoxF8ST4jTBix8G80XWIgB2uaiBpgp0CzJpYZ6rLgLVZ22LNt6zaVRBse3JSWdNotVZZkWxdZrdu9nGCriL5CwIRTpbLwafS14OgETEChtlQsFomA0vO664OyLLAtuxlmHoeQWiKMUHvDvknpDBGgKDQ7SZRtRQjTHVChdCH6+Lo9QSJI0ioSBuj7tKn/0wq+iWAcTUvp5G9Sr5oO/J+Wos8XTZvR2Svu1jVovo9fHB3JQT5nQNpnwxdoQyiQDgGKuCqi9xvRo0KUEXBbt2y94bnnn9tfr9fvw2i0XC7XcVg4Ch9SR25ujggnNJfhK86baFFUSLFgCUtgi40UnSvOmQU0YNAhmu3wuqjhYC3AMG090OTZqbnPAyKUtm7Z+mrPRTL2aiNJTYPCQAZKpcziPTgO2agAafSoNXmPu5Fx+Q15huhn+FylUho9ViN/49c1PEbHz/FZc8+Hha113T3W/Y4OhuEmW+K/efD7nN2PKAQw5Smi9xvRexVbt2y9/7nnn7vQtu0HFxYWVmEpnqBRe63AenI4kK4YyYdCCnODkP7I3NE3xcLNWb1DxrD9GLfmiagjQ3c1I9SIMLIOgcIVQ9xnTs10FPa9HLAdOlbDCCHU0Ke0B4VSqIkotMK30LSR0mDb28egPFdv8hPCR5C/0L9bf/qODWCZNqlqYzkOmKYNpuluwHHg345tQ73mkOPQnVCv4zNtg9Wwyd912wGrjs+NBbWaDafm+Br6qVNda+FdoZdNeYroKxBbt2w9iMIJAP66VqvtQl9Mp74nL1DIMCGFIJFw2RyJ5EPzGv6boSmowNOWmmKJMAKNCCEmmJigQxPbwvwCMSGK0EpQEHXR4K8VGH13zdYtW5VPSTye4J3x2LEKGE+fILwD9z71mkX4DW40qlULGpS/VComNBoW1Ewb6lXbrWgTIxp1Gyr1cJunTtHLgkkRfYUCd/bPPf/cBwDgTtu2r61UFojmhCNs1QfUXsrm0sITKJxSaHZLuVUlmAaEwsUrfJiwQg0L/90wG5Ek+Tpkw2R2mzDrxQMYEr51y9beiL5rg33flLY/2X7eh888P0OG7JgpB9s4i6SDr2BSRI8WSRA9Krxx9I3YrlUqlZCZXlcul3Gn/2mmOZBE1HRaaNALCgEiCISdMcxcLGK2EyDssNkfSZ6Nk24rDKiGPw4AF/TybU9PB2ujIhK9nGDLiN7TSILo/QRa8foa6rwn5jKsjoCv/VTMFQXSwsICVKuVsPeFD9yvMKGkEDkmenmJZ07V4PhMoC2Zb8uaTtDrlR8U0RWA9gg6HwDuZavBBFQXHVqlAmqBeB8okATcB5ruzi+VSveopyY2fB0fx16cOKZOvPjybNDDlWDyQBFdgQC7qpZKpeta2zMgY0dNAwWU2SM9m9wouxqUy2XyKmjO2OTv6lKppCLv4sWztEtzTwE3zZOHpjvxfz8k8v56vYMtI/q3JJhLYCDRUSglRfR+RqlU2lcul9H/+DUAuILdqjd6DX1QGMyAkXxJJmB7gdoQ+rFwjoKF5zRthy5f59mVg7+kd3pHkEaYSWF+vg6nyiacOlUNasnxQgmmFiiiKyxBqVTCZoN7yuXy1Ri519rPyiukSDsJGnXnpg/EI6hQENkkes8MG+7Nw400yEGFgicP5FMHAOBTlmnvThn6ud3OCK0tWAcUo4rxb9SwSVlQywGTduw2STSoRq0E7mfusW4KjEP7xTUabkI5Rik3OihE0IL9Oy9eLzSysx8EEyiiKywH1BKo9vR5jNxblt6Yk0ZK+LgWYZYIywSViKaDbsNKVwhh+HiI6gxBgRuZG6iAVpAHaOH5Hcyh/PG/vzk9WMqMYQGAZoqB7fIQi5TIAjCxTJZFO3E7bl4l40uSQfjGuV8EEyiiKywHqi3cUC6X9wHAFwDgWt5CuZqMvUSLYWWE2Ks7ThdYjrNYzqhZey/ewItnqEB6MM6LKnQOy3YePDlbu/bkrAxJCKGAUZ7Cg2n6STA1oYiu0AqqPWDe080AgOOqoIvEhEsMmk63QIF0s4q26yk86LdJ6hHs3XnxeuEBNX0pmBTRFdqBCqiry+XyZgC4npr4ejXq8SEabdfXGtIXb/rhkn9ftrtrS71MwE3ELvoM9irQCnHPxGNHhU9fa40A6hOiA3V69zrRI0mCfOThZ4Sc5/s/eETIeRLGahq9d503ik9ivEaZGo6e8iH9/OWXCTlPH/EooDTstRtaoLwpMg29XzUmoAt3qSK6gg+OeRj9ZiqcrpZMSB2k2hGORyWYj4I4nNcjm+gF+vw9SF8jteT0s2ACRXSFDvEq1VT30a/toQJqT8wbHPU8rCzspbU/2fO2KoK7X6Bjmv77GP030PfmPX+zY+fp62txb4j6XTCBIrpCCDxIxw3U5LeLPkM76BCF16hWdJA+q+p5WHm4x2Ml2UH51a5l+IWXz8x7eApQvgOe47zf7SmsBMEEiugKAoD0v58Ohh3U/LeZbnjYKwumYK/e5+UYFUSv0tf/8DxbCgEgykcqMdgmZWUCAP4/FPuqiCpgPLkAAAAASUVORK5CYII="/>
```

Comment:
A PNG in an SVG? what's the point?

Reply:
@AyzansAlt Yeah, I don't get the point here.

Reply:
Prolly because the code refers the SVG and @AyzansAlt didn't want to change that?

Reply:
Yeah, sorry for not responding earlier, I'm kinda leaving scratch and github for a white, and @locness3  is right

Reply:
Basically all i know is raster art :/

Reply:
That's okay! We'd just need to update the code a little (you can even do it!)

Reply:
yAy, I just thought that you wanted SVG art cause all I see is SVG art in all of @micahlt's projects